internal/runtimehandlerhooks/utils_test.go

```diff
@@ -4,10 +4,11 @@ import (
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/ginkgo/extensions/table"
 	. "github.com/onsi/gomega"
+	"k8s.io/kubernetes/pkg/kubelet/cm/cpuset"
 )
 
 var _ = Describe("Utils", func() {
-	Describe("UpdateIRQSmpAffinityMask", func() {
+	Describe("filterCPUs", func() {
```

Comment:
this function name changed, should probably change the test description that goes along with it

Reply:
totally right, missed it.

Reply:
done, thanks